README.md

```diff
@@ -1,2 +1,17 @@
 # k8s-service-stack-full
 temp repo to house the full service configuration while we split it
+
+
+# Notes for running on bare metal
+- Remove kube2iam from kube-system namespace
+- change elasticsearch-data-statefulset.yaml to use emptyDir instead of a persistentVolumeClaim:
+```
+      volumes:
+        - name: startup
+          configMap:
+            name: elasticsearch-startup-1
+        - name: elasticsearch-data
+          emptyDir: {}
```

Comment:
This is interesting and I think it warrants further discussion. 

So before, we where using persistent storage to provide and extra layer of protection to an already resilient cluster system which seems sensible. Recovering from a 'hardware' loss is usually an expensive event for a cluster.  If we change this to use local storage we lose this extra level and change the behaviour of our cluster but in theory the cluster should still be fine and we can take the hit.

However, I think we should consider setting DefaultStorageClass for the cluster on physical hardware so that we can mimic a kops cluster as closely as possible. That way when an engineer is dealing with the different k8s clusters there are less differences? Plus we also keep our extra level of protection. Thoughts?

Reply:
@tommyvn @cherwin

Reply:
sure - i was adding to this README to help anyone who builds a microdc on bare-metal in the meantime, to save them a bit of time troubleshooting.

We could look at rook.io to provide a platform-agnostic default storage class?